### ISSUE TITLE
Can the yolov7.weights be transformed to tensorrt ？

### ISSUE THREAD

**Issue by zxz-cc** [AlexeyAB/darknet#8680]
If something doesn’t work for you, then show 2 screenshots:
1. screenshots of your issue
2. screenshots with such information
```
./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/dog.jpg
 CUDA-version: 10000 (10000), cuDNN: 7.4.2, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2070
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
   0 conv     32       3 x 3/ 1    608 x 608 x   3 ->  608 x 608 x  32 0.639 BF
```

If you do not get an answer for a long time, try to find the answer among Issues with a Solved label: https://github.com/AlexeyAB/darknet/issues?q=is%3Aopen+is%3Aissue+label%3ASolved

**Comment by zxz-cc** [comment]
I have used the darknet framework to train a yolov7 model  and get the yolov7.weights 。I wonder if the model can be converted to tensorrt engine? like yolov7.weights ——>yolov7.onnx——>yolov7.engine?

**Comment by stephanecharette** [comment]
Did you google your question?  Or search in the existing Darknet issues?

**Comment by zxz-cc** [comment]
> 
Yes. But no answers were found at gooLe 。The same is ,i just found the yolov7.pt trained in pytorch can be converted to onnx , then yolov7.onnx can be converted to  tensorrt engine .
